src/main/java/randoop/contract/EqualsTransitive.java

```diff
@@ -0,0 +1,67 @@
+package randoop.contract;
+
+import randoop.Globals;
+
+/**
+ * The contract:
+ * <code>!(x0.equals(x1) &amp;&amp; x1.equals(x2))||x0.equals(x2)</code>.
+ */
+
+public class EqualsTransitive implements ObjectContract {
+    // Immutable, single instance
+    private static final EqualsTransitive instance = new EqualsTransitive();
+
+    // Private constructor
```

Comment:
This comment seems obvious to me.  I would omit it.

In addition, you should use /** ... */ Javadoc-style comments even for private members.  The reasons are:
 * there is a Javadoc mode for generating HTML that includes private members
 * IDEs will display Javadoc for private members when you hover/click on a use